### ISSUE TITLE
DOC: Fix up auto build procedure to upload Cheat Sheet to web site

### ISSUE THREAD

**Issue by Dr-Irv** [pandas-dev/pandas#40949]
Need to fix the procedures so that the PDFs (English, Japanese) for the cheat sheet here: https://github.com/pandas-dev/pandas/tree/master/doc/cheatsheet

get uploaded to here:
https://pandas.pydata.org/Pandas_Cheat_Sheet.pdf

(and when the Japanese one is there, can update the docs to link to it as well)

Note:  If we decide to move the location on the web site, then we should do a redirect, because there are links out there to the cheat sheet in the current location.

Followup to discussion here:  https://github.com/pandas-dev/pandas/issues/40168#issuecomment-819671740

**Comment by jorisvandenbossche** [comment]
For the website and docs, we rsync to the server for every commit on master. For example for the website:

https://github.com/pandas-dev/pandas/blob/a25198cb57e37f6f06b304c854750df18d57e224/.github/workflows/ci.yml#L124-L126

I assume the reason the `--exclude='Pandas_Cheat_Sheet*'` is there, is because the cheatsheet is not actually included in the built web sources, so when rsyncing that, the cheatsheet would otherwise be removed.

But adding an additional `rsync` call to synchronize the cheatsheets should be possible?

(cc @TomAugspurger @datapythonista)

**Comment by datapythonista** [comment]
Instead of another rsync, I think we can simply `cp` the cheat sheets from the repo to right location in the built web / docs after the web / docs are built. Does it make sense?

**Comment by jorisvandenbossche** [comment]
Yes, that's indeed even easier.

**Comment by Hallicopter** [comment]
I can give this a shot, seems like a good starter issue for a new contributor like me :)
From what I understand of the issue, I would have to make a change in the CI file pointed out here, so that as a part of the build pipeline the cheatsheet files are also `cp`'d in, correct?

I will have to figure out the Github workflows part, but I think I can give it a shot by the weekend. Is there anything else I should know?

**Comment by Hallicopter** [comment]
take

**Comment by Hallicopter** [comment]
> I think we can simply cp the cheat sheets from the repo to right location in the built web / docs after the web / docs are built. Does it make sense?

I went through the contribution guidelines and got a good sense of the overall build workflow. I am still a little confused as to where should I `cp` the cheatsheets to? Ie., where is the right location in the built web/docs that these cheatsheets can be copied to so that it shows up  https://pandas.pydata.org/Pandas_Cheat_Sheet.pdf

Sorry if I am asking obvious questions, but I am unfamiliar with the website build workflows for Pandas overall, so asking what comes to my mind.

Would adding the following solve the problem (again, unsure of the destination for the `cp`)
```
- name: Move cheatsheets into site directory
  run: cp doc/cheatsheets/Pandas_Cheat_Sheet* web/build/docs
```

(this would be added right above `name: Save website as an artifact`)


Correct me if I have misunderstood anywhere.

**Comment by jorisvandenbossche** [comment]
> I am still a little confused as to where should I `cp` the cheatsheets to? Ie., where is the right location in the built web/docs that these cheatsheets can be copied to so that it shows up https://pandas.pydata.org/Pandas_Cheat_Sheet.pdf

Since the cheatsheet uses a "top-level" URL (it lives directly under pandas.pydata.org), I think it should be copied to `web/build/`, as that is the main directory that is synced with the server for the main website.

> Would adding the following solve the problem

Yes, that seems correct to me (with `web/build` as target then)

@Hallicopter Thanks for working on it!

**Comment by datapythonista** [comment]
Not sure, but just came to my mind that maybe we have a nginx rule for the cheatsheets. I remember we had something being served from a different place, and would make sense that are the cheatsheets, otherwise they could be overwritten when updating the web/docs.

I'm in a bus the whole day, so can't check until tomorrow. But if that's the case we'll have to synchronize merging the PR with updating nginx.

**Comment by Hallicopter** [comment]
Yep, no problem! If I were to locally test my changes, is there any suggestion you have? I can just build the website locally using the commands I am assuming. 
Also, if there are some other nuances regarding the nginx configs, do let me know!

**Comment by datapythonista** [comment]
I was just checking, there is nothing in the nginx config. It's just the exclude in the rsync that is being used to avoid them being overwritten.

I'm +1 on moving the cheatsheets to their own directory (e.g. `cheatsheets/`).

In any case, your command seems right (I'd just add `Copy` instead of `Move` in the title since that's what you're doing). And then you'll have to remove the `exclude` of the cheatsheets in the sync.

**Comment by Hallicopter** [comment]
That's great! I shall put in a PR in a few days for review.

**Comment by Dr-Irv** [comment]
@Hallicopter any update on doing a PR for this?
